build.gradle

```diff
@@ -56,9 +63,9 @@ ext {
     compileSdkVersion = 30
     minSdkVersion = 24
     targetSdkVersion = 30
-
-    awsSdkVersion = '2.52.1'
     awsKotlinSdkVersion = '0.17.6-beta'
+    awsSDKReleaseVersion = '2.53.0'
+    awsSdkVersion = useAwsSdkReleaseBuild() ? awsSDKReleaseVersion : "latest.integration"
```

Comment:
dev-preview does not need SDK snapshot build. Should we remove this?

Reply:
@sdhuka

Reply:
Yes, this could be removed from dev-preview.